validators/smoke/genesis-chain/test.sh

```diff
@@ -2,7 +2,7 @@
 set -e
 
 # Simply query the 5th block and check that it's correct
-reply=`curl -s -X POST -H "Content-Type: application/json" --data '{"jsonrpc":"2.0","method":"eth_getBlockByNumber","params":["0x5", false],"id":0}' "$HIVE_CLIENT_IP:8545"`
+reply=`curl -s -X POST -H "Content-Type: application/json" --data '{"jsonrpc":"2.0","method":"eth_getBlockByNumber","params":["0x5", false],"id":1}' "$HIVE_CLIENT_IP:8545"`
```

Comment:
I understand that. It's just that since the consensus tests start with id=1 on RPC requests, I thought it'd be desirable to do the same here. I'll revert these changes and fix pyethapp, though